pkg/addr/isdas.go

```diff
@@ -232,6 +245,8 @@ func ParseIA(ia string) (IA, error) {
 
 // MustParseIA parses s and returns the corresponding addr.IA object. It
 // panics if s is not a valid ISD-AS representation.
+// @ requires false
+// @ decreases
```

Comment:
```suggestion
```

We don't care for termination of functions that we can't call

Reply:
Done.